src/view/screens/Home.tsx

```diff
@@ -80,14 +112,37 @@ function HomeScreenReady({
   const pagerRef = React.useRef<PagerRef>(null)
   const lastPagerReportedIndexRef = React.useRef(selectedIndex)
   React.useLayoutEffect(() => {
+    let initialIndex = selectedIndex
+    if (currentStarterPack?.initialFeed) {
+      if (currentStarterPack.initialFeed === 'following') {
+        initialIndex = allFeeds.findIndex(f => f === 'following')
+      } else {
+        initialIndex = allFeeds.findIndex(
+          f => f === `feedgen|${currentStarterPack.initialFeed}`,
+        )
+      }
+      if (initialIndex === -1) {
+        initialIndex = 0
+      }
+      setCurrentStarterPack(undefined)
+    }
+
```

Comment:
note that this is a hack and needs work. it doens't actually work right now, though it worked on the local dev env. not sure why, maybe something with how we add discover?